tests/testthat/test-install-torsten.R

```diff
@@ -0,0 +1,157 @@
+## Adapted from test-install.R from the cmdstanr package
+
+context("install_torsten")
+
+torsten_test_tarball_url <- Sys.getenv("TORSTEN_TEST_TARBALL_URL")
+if (!nzchar(torsten_test_tarball_url)) {
+  torsten_test_tarball_url <- NULL
+}
+
+test_that("install_torsten() successfully installs torsten", {
```

Comment:
-   We've got a helper in bbr (`skip_if_over_rate_limit`) that skips the test if we've exhausted the GitHub API limits. bbr.bayes should be able to import that for use on its end (see `tests/testthat/setup-imports.R`).

-   It's not too uncommon to add a flag to skip network-dependent tests, but we don't have anything like that in bbr or bbr.bayes yet. We might want to eventually do that for these tests, but for now, let's at least mark the tests that actually do the installation as "long" tests by adding something like
    
    ```
    skip_long_tests("long-running install_torsten() tests")
    ```